yt_dlp/extractor/tv8.py

```diff
@@ -0,0 +1,82 @@
+from .common import InfoExtractor
+from .skyit import TV8ItIE
+from ..utils import clean_html, traverse_obj, url_or_none, urljoin
+
+
+class TV8StreamingIE(InfoExtractor):
+    IE_NAME = 'TV8Streaming'
+    IE_DESC = 'TV8 Live'
+    _VALID_URL = r'https?://(?:www\.)?tv8\.it/streaming'
+    _TESTS = [{
+        'url': 'https://tv8.it/streaming',
+        'info_dict': {
+            'id': 'tv8',
+            'ext': 'mp4',
+            'title': str,
+            'description': str,
+            'is_live': True,
+            'live_status': 'is_live',
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = 'tv8'
+        streaming = self._download_json(
+            'https://tv8.it/api/getStreaming', video_id, 'Downloading streaming data', fatal=False)
+        livestream = self._download_json(
+            'https://apid.sky.it/vdp/v1/getLivestream?id=7', video_id, 'Downloading manifest json info')
+
+        return {
+            'id': video_id,
+            'is_live': True,
+            'formats': self._extract_m3u8_formats(livestream['streaming_url'], video_id),
+            **traverse_obj(streaming, ('info', {
+                'title': ('title', 'text', {str}),
+                'description': ('description', 'html', {clean_html}),
+            })),
+        }
+
+
+class TV8PlaylistIE(InfoExtractor):
+    IE_NAME = 'TV8Playlist'
+    IE_DESC = 'TV8 Playlist'
+    _VALID_URL = r'https?://(?:www\.)?tv8\.it/(?!video)[^/#?]+/(?P<id>[^/#?]+)'
+    _TESTS = [{
+        'url': 'https://tv8.it/intrattenimento/tv8-gialappas-night',
+        'playlist_mincount': 32,
+        'info_dict': {
+            'id': 'tv8-gialappas-night',
+            'title': 'Tv8 Gialappa\'s Night',
+            'description': 'md5:c876039d487d9cf40229b768872718ed',
+            'thumbnail': 'https://static.sky.it/editorialimages/47b87cd71c2a4b71c4acbb8be04ae65dd71ce7ff/tv8/assets/entertainment/tv8-gialappa\'s-night/Gialappa\'sNight%20sito.jpg?auto=webp&im=Resize,width=1040,height=1040',
```

Comment:
Let's use regex matching to shorten these thumbnail lines
```suggestion
            'thumbnail': r're:^https://static\.sky\.it/.+\.(png|jpe?g|webp)',
```